### ISSUE TITLE
Saving KYC process

### ISSUE THREAD

**Issue by Nathael** [blue-horizon-lab/connecty-ico#58]
Once the user has started filling the KYC form, he can save the form content.
This moves the status to "Initiated"
Displaying the KYC block will recall the previously saved content, until the form is sent for validation.

**Comment by slamazelo** [comment]
Modified : the data is still saved in db from the beginning to the end of the process

**Comment by Nathael** [comment]
No, no data is saved in database while the user is filling a form.
Saving the form content allows the user to recall the content for future completion without submitting it for review.

**Comment by slamazelo** [comment]
"recall the content for future completion" means "being saved in some database somewhere". so, we must save in db from the beginning or give me the solution.

**Comment by Nathael** [comment]
"save in db from the beginning" is ambiguous.

The save in the database happens when the user klicks the "save" button, not before.
Thus nothing is saved "from the begining"
If the user does not click on the save button and goes to another part of the dashboard, the data is lost, and should not have been sent to the server, ever.

This means that the KYC form has two submit buttons : 
- one which only saves the content to the database for future completion, but does not submit it for review by blue-horizon KYC team
- One which submits the KYC content for review, which means that the content cannot be modified any-more until the KYC team has reviewed it, and may not be modified any more at all in fact, as the team will either validate, refuse, or ask more info/data, but this data will be kept as-is.

This also means that we can add a popup when the user has some form fields filled and he leaves the KYC module, to let him know that he can save for future completion, or leave the KYC module and loose the form content.
